packages/sdk-js/test/cases.json

```diff
@@ -1317,13 +1317,21 @@
     ]
   ],
   "hash": [
-    ["a", 0.22],
-    ["b", 0.077],
-    ["ab", 0.946],
-    ["def", 0.652],
-    ["8952klfjas09ujkasdf", 0.549],
-    ["123", 0.011],
-    ["___)((*\":&", 0.563]
+    ["", "a", 1, 0.22],
+    ["", "b", 1, 0.077],
+    ["b", "a", 1, 0.946],
+    ["ef", "d", 1, 0.652],
+    ["asdf", "8952klfjas09ujk", 1, 0.549],
+    ["", "123", 1, 0.011],
+    ["", "___)((*\":&", 1, 0.563],
+    ["seed", "a", 2, 0.0505],
+    ["seed", "b", 2, 0.2696],
+    ["foo", "ab", 2, 0.2575],
+    ["foo", "def", 2, 0.2019],
+    ["89123klj", "8952klfjas09ujkasdf", 2, 0.124],
+    ["90850943850283058242805", "123", 2, 0.7516],
+    ["()**(%$##$%#$#", "___)((*\":&", 2, 0.0128],
+    ["abc", "def", 99, -1]
```

Comment:
Since we are allowing the user to choose between hashing versions, shouldn't we keep tests for v1 to make sure we don't break anything for v1 when moving forward?

Reply:
Wait, never mind, it looks like the 3rd positional argument may be the hash version.